tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/phases/TornadoTaskSpecialisation.java

```diff
@@ -194,8 +193,7 @@ private void printWarningMessageForDynamicLoopBounds() {
     }
 
     private void evaluate(final StructuredGraph graph, final Node node, final Object value) {
-        if (node instanceof ArrayLengthNode) {
-            ArrayLengthNode arrayLength = (ArrayLengthNode) node;
+        if (node instanceof ArrayLengthNode arrayLength) {
```

Comment:
there is no need to add a variable name if there is no usage.

Reply:
it is used further down, lines 217-218